utils/net.py

```diff
@@ -43,13 +42,7 @@ def my_ip_address(http=False):
 
     """
     # Allow for DockerBot to set outside IP
-    try:
-        # Check the environment first
-        return os.environ['CFME_MY_IP_ADDRESS']
-    except KeyError:
-        # Fall back to having an appliance tell us what it thinks our IP
-        # address is
-        return store.current_appliance.ssh_client().client_address()
+    return store.current_appliance.my_ip_address
```

Comment:
Should we be tying our IP to the appliance instance? What happens if we push a new appliance onto the stack? In fact, shouldn't this just be ```store.my_ip_address```?